test/schema/levels.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+module.exports = {
```

Comment:
There are no changes in this file except converting it from JSON into JavaScript.